lib/v2/screens/import_key/components/import_key_accounts_widget.dart

```diff
@@ -26,7 +27,9 @@ class ImportKeyAccountsWidget extends StatelessWidget {
                           padding: const EdgeInsets.all(16),
                           child: InkWell(
                             borderRadius: BorderRadius.circular(defaultCardBorderRadius),
-                            onTap: () => () {},
+                            onTap: () {
+                              context.read<ImportKeyBloc>().add(AccountSelected(account: profile.account));
```

Comment:
handle event when user selects and account

Reply:
Nice i forget bloc has extencions for a shorter sintax